### DISCUSSION TITLE
nodes shape should accept a list

### DISCUSSION THREAD

**Discussion by ggrrll** [networkx/networkx#4677] [Ideas]
Hi, 

I think that would be great to have `node_shape` taking a list / dict (in `nx.draw` ), pretty much  like it happens with colors, positions and sizes

Thanks 

-- similar to https://github.com/networkx/networkx/issues/1289

**Comment by jpmedras** [comment]
I agree with you. It can be useful to a allow an array of shapes instead of just one shape for all nodes, it would be similar to `node_color` parameter.
